apis/components/v1alpha1/groupversion_info.go

```diff
@@ -26,7 +26,7 @@ import (
 
 var (
 	// GroupVersion is group version used to register these objects
-	GroupVersion = schema.GroupVersion{Group: "components.opendatahub.io", Version: "v1"}
+	GroupVersion = schema.GroupVersion{Group: "platform.opendatahub.io", Version: "v1alpha1"}
```

Comment:
wonder if we want to have an additional level i.e `components.platform.opendatahub.io`, `services.platform.opendatahub.io`, and so on. 

no strong opinion though

Reply:
so we might have different version on component and service later? v1alpha1 v1beta10

Reply:
Mainly for discovering:

to know what components are shipped as part of the platform:
```
kubectl api-resources --api-group components.platform.opendatahub.io
```

to know what services  are shipped as part of the platform:
```
kubectl api-resources --api-group services.platform.opendatahub.io
```

etc

Reply:
Yeah we probably can have different versions. And yes I remember we wanted to have easy discovery, so having this 2 groups makes sense.